scripts/size_report/size_report_helper.ts

```diff
@@ -77,7 +77,7 @@ export function upload(
     console.log(`Response status code: ${response.statusCode}`);
     response.on('data', console.log);
     response.on('end', () => {
-      if (response.statusCode !== 202) {
+      if (response.statusCode !== 200 && response.statusCode !== 202) {
```

Comment:
Should we still accept 202?

Reply:
Yes, for now. The api for the old size report still returns 202. I'm planning to migrate it to 200 as well. I'll remove the 202 here (and a few other places in android repo) when that happens.